tests/helpers.py

```diff
@@ -43,7 +43,8 @@ def write_unprocessed_events(
 
 
 def write_raw_unprocessed_events(
-    storage: WritableStorage, events: Sequence[Union[InsertEvent, Mapping[str, Any]]]
+    storage: WritableStorage,
+    events: Sequence[Union[InsertEvent, Mapping[str, Any], Any]],
```

Comment:
Any chance we can do better than `Any` for the message type?

Reply:
Ended up using `write_unprocessed_events` instead and reverted the typing here.